### PR TITLE
change in pubspec files

### PR DESCRIPTION
**Pull request by bhargavraviya**
Flutter Upgrade from scratch

- change pubspec new packages
- sdk version changes for ^3.5.3
- change minor text changes

please check it and merge @YC

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,7 +30,7 @@
 .packages
 .pub-cache/
 .pub/
-build/
+/build/
```
Comment (YC) [inline .gitignore]: `build/` should cover `/build/`. 
Was this from `flutter migrate`?

File: analysis_options.yaml
Comment (YC) [inline analysis_options.yaml:1]: Files should have trailing newlines. 
Was this from `flutter migrate`?

File: pubspec.yaml [hunk 2]
```diff
@@ -1,11 +1,10 @@
 name: another_authenticator
-description: Another two-factor authentication app.
-publish_to: none
-
+description: "Another two-factor authentication app."
+publish_to: 'none'
```
Comment (YC) [inline pubspec.yaml:3]: Inconsistent quotes. 
But looks like this is consistent with template. 
~~Was this automatic, or manually edited?~~

File: .gitignore
Comment (YC) [inline .gitignore:1]: Files should have trailing newlines. 
I need to add an editorconfig for this project :)

File: pubspec.yaml
Comment (YC) [inline pubspec.yaml:1]: Missing trailing newline

**Review by YC — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (bhargavraviya) [reply .gitignore]: When you create a new Flutter project, these files generate automatically, including the .gitignore file.

File: pubspec.yaml
Comment (bhargavraviya) [reply pubspec.yaml:1]: When you create a new Flutter project, these files generate automatically, including the .gitignore file.

File: .gitignore (hunk 1 above)
Comment (YC) [reply .gitignore]: This actually has to be `build/` because of nested projects.

Comment (YC) [reply .gitignore]: The templates have trailing newlines.

File: pubspec.yaml
Comment (YC) [reply pubspec.yaml:1]: The templates have trailing newlines.

**Review by YC — APPROVED** [review]

**Pull request merged**